core/test/integration/api/api_oembed_spec.js

```diff
@@ -0,0 +1,69 @@
+const common = require('../../../server/lib/common');
+const nock = require('nock');
+const OembedAPI = require('../../../server/api/oembed');
+const should = require('should');
+
+describe('Oembed API', function () {
```

Comment:
What was the decision for adding these tests as integration test? These tests look like unit tests to me.

Reply:
All of the other API endpoints only seemed to have integration tests so I didn't know any different 😄

Reply:
Yeah, none of them were moved or reconsidered yet (time problem). But the goal is to test e.g. error cases, edge cases in a unit tests for any module. And 1-2 routing tests to ensure the whole flow is covered :)